python/pyspark/sql/tests/connect/streaming/test_parity_streaming.py

```diff
@@ -22,32 +22,7 @@
 
 
 class StreamingParityTests(StreamingTestsMixin, ReusedConnectTestCase):
-    @unittest.skip("Query manager API will be supported later with SPARK-43032.")
-    def test_stream_await_termination(self):
-        super().test_stream_await_termination()
-
-    @unittest.skip(
-        "Query immediately quits after throw, "
-        + "allowing access to supported queries will be added in SPARK-42962."
-    )
-    def test_stream_exception(self):
```

Comment:
Do you know why this was here before?

Reply:
Yes because it access stopped query. We seem to have a slight conflict with spark tasks (42962 and the real better session mgmt one). Anyway it should be fine after this PR